Welcome page links, CF Actuator Integration (#158)
SteeltoeOSS/Documentation: api/v3/management/cloud-foundry.md

See the [HTTP Access](./using-endpoints.md#http-access) section to see the overall steps required to enable HTTP access to endpoints in an ASP.NET Core application.

The Cloud Foundry actuator, it's security middleware and CORS policy can be added to the application in either `program.cs` or `startup.cs` and either specifically or in a grouping with all the other actuators. All of these options can function effectively the same way, so this may be interpreted as a style choice.

>Of all the options provided, the approach of using `AddAllActuators()` on the `HostBuilder` is most recommended.

### Program.cs

If you prefer maximum convenience, extension methods for both `IHostBuilder` and `IWebHostBuilder` can configure all actuators at once with a single line of code:

```csharp
using Steeltoe.Management.CloudFoundry; // for .AddCloudFoundryActuators()
using Steeltoe.Management.Endpoint;     // for other options
...
    public static void Main(string[] args)
    {
        BuildWebHost(args).Run();
    }
    public static IHost BuildHost(string[] args) =>
        Host.CreateDefaultBuilder(args)

>             .AddCloudFoundryActuators() // add CF + security + others, deprecated in 3.1.0

Author: This comment was added inside the context of a commit... moved the conversation over to the files tab so it can carry across commits

| `ApplicationId` | The ID of the application used in permissions check. | VCAP settings |
| `CloudFoundryApi` | The URL of the Cloud Foundry API. | VCAP settings |

>Each setting in the preceding table must be prefixed with `Management:Endpoints:CloudFoundry`.

## Enable HTTP Access

The default path to the Cloud Foundry endpoint is computed by combining the global `Path` prefix setting together with the `Id` setting described in the table above. The global path is always set to `/cloudfoundryapplication`. Apps Manager expects this endpoint to be available at `/cloudfoundryapplication`.

See the [HTTP Access](./using-endpoints.md#http-access) section to see the overall steps required to enable HTTP access to endpoints in an ASP.NET Core application.

> The Cloud Foundry actuator, the Cloud Foundry actuator's security middleware and CORS policy can be added to the application in either `program.cs` or `startup.cs` and either specifically or in a grouping with all the other actuators. All of these options can function effectively the same way, so this may be interpreted as a style choice.

Reviewer: ```suggestion
The Cloud Foundry actuator, the actuator's security middleware, and the CORS policy can be added to the application in either `program.cs` or `startup.cs`. This can be done individually or in a grouping with all the other actuators. All of these options will function effectively the same way, so this is a style choice.
```

>Of all the options provided, the approach of using `AddAllActuators()` on the `HostBuilder` is most recommended.

### Program.cs

If you prefer maximum convenience, extension methods for both `IHostBuilder` and `IWebHostBuilder` can configure all actuators at once with a single line of code:

```csharp
using Steeltoe.Management.CloudFoundry; // for .AddCloudFoundryActuators()
using Steeltoe.Management.Endpoint;     // for other options
...
    public static void Main(string[] args)
    {
        BuildWebHost(args).Run();
    }
    public static IHost BuildHost(string[] args) =>
        Host.CreateDefaultBuilder(args)

>             .AddAllActuators()            // add CF + security (when deployed to CF) and all others

Author: > @jkonicki: I find adding all these methods confusing. I recommend having the preferred method uncommented and the rest of them commented out with individual comments indicating when to use them.

Is this better?

See the [HTTP Access](./using-endpoints.md#http-access) section to see the overall steps required to enable HTTP access to endpoints in an ASP.NET Core application.

> The Cloud Foundry actuator, the Cloud Foundry actuator's security middleware and CORS policy can be added to the application in either `program.cs` or `startup.cs` and either specifically or in a grouping with all the other actuators. All of these options can function effectively the same way, so this may be interpreted as a style choice.

Reviewer: Remove the second "the Cloud Foundry's actuator.   The sentence starts with it already.  For example:
"The Cloud Foundry actuator's security middleware and CORS policy can be added to the application in either `program.cs` or `startup.cs` specifically or in a grouping with all the other actuators."

    public IConfiguration Configuration { get; }
    public void ConfigureServices(IServiceCollection services)
    {
        // for versions >= 3.1.0-rc1, add all actuators and Cloud Foundry integration pieces
        services.AddAllActuators(Configuration);

> 

Reviewer: This looks much better
